pkg/logs/decoder/line_handler.go

```diff
@@ -170,7 +170,15 @@ func (h *MultiLineHandler) run() {
 				return
 			}
 			// process the new line and restart the timeout
-			flushTimer.Stop()
+			if !flushTimer.Stop() {
+				// stop doesn't not prevent a tick from the Timer if they happen at the same time
```

Comment:
if it happens

Reply:
they was Stop and tick, but that might not be super clear, will change it with my next PR